### PR TITLE
Rel 7.0.0 min env

### PR DESCRIPTION
**Pull request by sedin-tushar**
# SunbirdEd - Portal
---

### Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
---

#### Example
- [x] Applicable
- [ ] Not applicable

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/routes/contentRoutes.js [hunk 1]
```diff
@@ -5,7 +5,8 @@
  */
 const proxyUtils = require('../proxy/proxyUtils.js')
 const envHelper = require('../helpers/environmentVariablesHelper.js')
-const contentURL = envHelper.CONTENT_URL
+const utils = require('../helpers/utils.js');
+const contentURL  = utils.defaultHost(utils.envVariables.CONTENT_URL);
```
Comment (rajeevsathish) [inline src/app/routes/contentRoutes.js:9]: please add checks here
utils?.defaultHost(utils?.envVariables?.CONTENT_URL);

File: src/app/routes/discussionsForum.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 const proxyUtils = require('../proxy/proxyUtils.js');
 const BASE_REPORT_URL = "/discussion";
 const proxy = require('express-http-proxy');
-const { discussions_middleware } = require('../helpers/environmentVariablesHelper.js');
+const utils = require('../helpers/utils.js');
+const  discussions_middleware = utils.defaultHost(utils.envVariables.discussions_middleware);
```
Comment (rajeevsathish) [inline src/app/routes/discussionsForum.js:5]: Please add checks
utils?.defaultHost(utils?.envVariables?.discussions_middleware);

File: src/app/routes/groupRoutes.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 const proxyUtils = require('../proxy/proxyUtils.js')
 const envHelper = require('../helpers/environmentVariablesHelper.js')
-const learnerURL = envHelper.LEARNER_URL
+const utils = require('../helpers/utils.js');
+const learnerURL  = utils.defaultHost(utils.envVariables.LEARNER_URL);
```
Comment (rajeevsathish) [inline src/app/routes/groupRoutes.js:4]: Please add checks
const learnerURL = utils?.defaultHost(utils?.envVariables?.LEARNER_URL);

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/routes/contentRoutes.js (hunk 1 above)
Comment (sedin-tushar) [reply src/app/routes/contentRoutes.js]: Added Safety Checks

File: src/app/routes/discussionsForum.js (hunk 2 above)
Comment (sedin-tushar) [reply src/app/routes/discussionsForum.js]: added safety checks for all places with utils.defaultHost

File: src/app/routes/groupRoutes.js (hunk 3 above)
Comment (sedin-tushar) [reply src/app/routes/groupRoutes.js]: added safety checks for all places with utils.defaultHost

**Review by vinukumar-vs — DISMISSED** [review]

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**